k8sclient/k8sclient.go

```diff
@@ -624,3 +634,47 @@ func GetDefaultNetworks(k8sArgs *types.K8sArgs, conf *types.NetConf, kubeClient
 
 	return nil
 }
+
+func getPodDefaultNetworkAnnotation(client KubeClient, k8sArgs *types.K8sArgs) (string, error) {
+	logging.Debugf("getPodDefaultNetworkAnnotation: %v, %v", client, k8sArgs)
+	pod, err := client.GetPod(string(k8sArgs.K8S_POD_NAMESPACE), string(k8sArgs.K8S_POD_NAME))
+	if err != nil {
+		return "", logging.Errorf("getPodDefaultNetworkAnnotation: failed to query the pod %v in out of cluster comm: %v", string(k8sArgs.K8S_POD_NAME), err)
+	}
+
+	if v, ok := pod.Annotations["multus-cni.io/default-network"]; ok {
+		return v, nil
+	} else {
+		return "", nil
+	}
+}
+
+// tryLoadK8sPodDefaultNetwork get pod default network from annotations
+func tryLoadK8sPodDefaultNetwork(k8sArgs *types.K8sArgs, conf *types.NetConf, kubeClient KubeClient) (*types.DelegateNetConf, error) {
+	logging.Debugf("tryLoadK8sPodDefaultNetwork: %v, %v", kubeClient, k8sArgs)
+
+	netAnnot, err := getPodDefaultNetworkAnnotation(kubeClient, k8sArgs)
+	if err != nil {
+		return nil, err
```

Comment:
May need to replace 'logging.Errorf()' with err why tryLoadK8sPodDefaultNetwork() is failed, like
```
   return nil, logging.Errorf("<add error messsage here>: %v", err)
```